lib/main_common.pm

```diff
@@ -612,7 +612,12 @@ sub load_filesystem_tests {
 
 sub load_wicked_tests {
     loadtest "console/wicked_before_test";
-    loadtest "console/wicked_basic";
+    if (check_var('WICKED', '1')) {
```

Comment:
Does this really make sense? Why should we call "wicked_risky" if !WICKED?

Reply:
probably we can rename it to "basic" ? does it make more sense then ?

Reply:
Like check_var ('WICKED', 'basic') maybe

Reply:
Then you can actually call loadtest 'wicked_' . get_required_var('WICKED');